Makefile

```diff
@@ -0,0 +1,39 @@
+.PHONY: default
+default: help
+
+.PHONY: up
+up: ## Build and start containers
+	mkdir -p node_modules
+	mkdir -p dist
+	docker compose up -d --build
+
+.PHONY: build
```

Comment:
The build is over-caching currently. This is probably due to the dockerfile, but that is why the build has --no-cache.... it's a bug, but kind of not important right now.